src/main/java/uk/gov/hmcts/divorce/caseworker/service/notification/FinalOrderGrantedNotification.java

```diff
@@ -135,14 +133,14 @@ private Map<String, String> solicitorTemplateContent(final CaseData caseData,
     }
 
     private Map<String, String> citizenTemplateContent(final CaseData caseData,
-                                                         final Long caseId,
-                                                         final Applicant applicant,
-                                                         final Applicant partner) {
+                                                       final Long caseId,
+                                                       final Applicant applicant,
+                                                       final Applicant partner) {
         Map<String, String> templateVars =
             commonContent.mainTemplateVars(caseData, caseId, applicant, partner);
 
         //TODO: temporarily set to false, need to update when Final Order switch to sole journey is being developed
```

Comment:
this comment needs update ?

Reply:
Discussed - Happy to keep TODO for now as the functionality it describes hasn't been implemented yet.